src/ipc/screenshot.ts

```diff
@@ -0,0 +1,10 @@
+import { IpcMain, IpcMainInvokeEvent } from 'electron';
+
+export const channel = 'screenshot';
+
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(channel, async (event: IpcMainInvokeEvent) => {
+    const nativeImage = await event.sender.capturePage();
+    return nativeImage.toPNG();
```

Comment:
PNG seemed like a safe format but we could make other choices here.

Reply:
Seems fine to start!